tools/dngmeta.sh

```diff
@@ -1,178 +0,0 @@
-#!/bin/bash
```

Comment:
I think we want to keep this tool, just remove the adobe_coeff related bits. We still want the rawspeed XML output.

Reply:
Done.